visualization/CMakeLists.txt

```diff
@@ -61,6 +61,12 @@ if(VTK_RENDERING_BACKEND_OPENGL_VERSION VERSION_LESS 2)
     "src/vtk/vtkVertexBufferObject.cxx"
     "src/vtk/vtkVertexBufferObjectMapper.cxx"
   )
+
+  # Code in this module may use deprecated declarations when using OpenGLv1
+  # Add the GCC exclusive rules for -Werror only for OpenGLv1 compile to avoid build interruption
+  if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
Missing check for OpenGL version here?

Reply:
Thanks for your reviewing!
https://github.com/PointCloudLibrary/pcl/blob/b5d806c0b6b214d23c0d3cb486714175e07e0852/visualization/CMakeLists.txt#L59-L70
I put this block in `if` scope, and L59 will do the check.